src/main/java/uk/gov/hmcts/reform/locationrefdata/camel/mapper/BuildingLocationMapper.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.locationrefdata.camel.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.data.ingestion.camel.mapper.IMapper;
+import uk.gov.hmcts.reform.locationrefdata.camel.binder.BuildingLocation;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.locationrefdata.camel.util.LrdLoadUtils.trim;
+
+@Component
+public class BuildingLocationMapper implements IMapper {
+
+    @Override
+    public Map<String, Object> getMap(Object buildingLocationObj) {
+        BuildingLocation buildingLocation = (BuildingLocation) buildingLocationObj;
+        Map<String, Object> buildingLocationParamMap = new HashMap<>();
+        buildingLocationParamMap.put("epimms_id", trim(buildingLocation.getEpimmsId()));
+        buildingLocationParamMap.put("building_location_name", trim(buildingLocation.getBuildingLocationName()));
+        buildingLocationParamMap.put("building_location_status", trim(buildingLocation.getBuildingLocationStatus()));
+        buildingLocationParamMap.put("area", trim(buildingLocation.getArea()));
```

Comment:
may be we can use @@BindyConverter rather calling trim every places.one time class put on binder class

Reply:
the trim() is not applied on all the fields in the binder object. Can we apply this on selective fields?